main.go

```diff
@@ -113,8 +142,8 @@ type Tag struct {
 type TimeEntry struct {
 	Wid         int        `json:"wid,omitempty"`
 	ID          int        `json:"id,omitempty"`
-	Pid         int        `json:"pid"`
-	Tid         int        `json:"tid"`
+	Pid         *int       `json:"pid"`
```

Comment:
Adding nullability to prevent Bad Request with project or task with ID 0.

Reply:
I need to double-check this one because it will add a lot of boilerplate in `alfred-toggl` to check non-nil.

Reply:
Should this still use `pid` and `tid`, or switch to the new `project_id` and `task_id` fields?

Reply:
I deliberately didn't migrate to new fields yet where it was not needed. I will do it once everywhere in separate PR.

Reply:
They need to be nullable otherwise updates fail on on-accessible projects or tasks. BTW you were right, we need to move to `project_id` and `task_id` already.

Anyway, I updated the rest of the structs, where it was possible or where I could confirm it, to use new fields too.